### ISSUE TITLE
Release Pillow 5.1.0 on April 1, 2018

### ISSUE THREAD

**Issue by radarhere** [python-pillow/Pillow#3018]

**Comment by aclark4life** [comment]
No joke! @wiredfool Are you on this one?

**Comment by wiredfool** [comment]
Yeah, release at some point.  We'll see if it gets tagged tonight. There are likely other things that need release notes and documentation as well as the one that's tagged.

**Comment by aclark4life** [comment]
@wiredfool Don't miss your chance to do a Pillow release on Easter Sunday AND April Fool's Day (has happened since 1956: http://www.bbc.com/news/uk-43611147).

**Comment by wiredfool** [comment]
Not going to happen tonight.

**Comment by aclark4life** [comment]
@wiredfool NP, let's try to make the next one in 2080 then… by then, assuming +1 major releases per year we may gone, but Pillow 67.0.0 should be released on time…

**Comment by wiredfool** [comment]
Release is tagged...

**Comment by cgohlke** [comment]
[Windows binaries](https://www.lfd.uci.edu/~gohlke/download/Pillow-5.1.0-win-dist.zip)

**Comment by wiredfool** [comment]
Thanks @cgohlke . Binaries are uploaded, RTD is updated. I haven't hidden the old release on pypi, as that's not in the new interface, so I think it's time to drop that task.

**Comment by aclark4life** [comment]
Thanks @wiredfool @cgohlke et al., I just hid 5.0.0 on PyPI

**Comment by aclark4life** [comment]
Also tweeted: https://twitter.com/PythonPillow/status/980918773283778560
